src/services/web3/utils/helpers.ts

```diff
@@ -0,0 +1,44 @@
+import { ExternalProvider, Web3Provider } from '@ethersproject/providers';
+import ConfigService from '@/services/config/config.service';
+
+export async function importPolygonDetailsToWallet(provider: ExternalProvider) {
+  const configService = new ConfigService();
+  const polygonNetworkConfig = configService.getNetworkConfig('137');
```

Comment:
should I pull this chainId '137' out of the configs constant?

Reply:
There is actually a `Network.POLYGON` (`network.ts`) which you can use. I guess I should change the name to something like `ChainId.POLYGON` so it will be clearer that its a chain id.